apps/web/src/routes/_public/route.tsx

```
@@ -26,6 +26,11 @@ function RouteComponent() {
                 Pricing
               </Link>
             </Button>
+            <Button variant="link" asChild>
+              <a href="https://support.tryfrontdesk.app" target="_blank">
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)